docs/source/tutorial_i.ipynb

```diff
@@ -365,7 +365,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "![title](figures/Conditional_Distribution_of_Data.png)"
+    "![title](figures / Conditional_Distribution_of_Data.png)"
```

Comment:
What are these changes? Does this still render the embedded image correctly?

Reply:
That's not supposed to be there. Much be a linting setting in my IDE. Frustrating.

Reply:
To answer your question, no it doesn't render the image correctly.